pkg/cloud/services/ec2/launchtemplate.go

```diff
@@ -277,5 +280,48 @@ func (s *Service) SDKToLaunchTemplate(d *ec2.LaunchTemplateVersion) (*expinfrav1
 		i.NetworkInterfaces = append(i.NetworkInterfaces, *tmp)
 	}
 
+	for _, id := range v.SecurityGroupIds {
+		i.AdditionalSecurityGroups = append(i.AdditionalSecurityGroups, infrav1.AWSResourceReference{ID: id})
+	}
+
 	return i, nil
 }
+
+// launchTemplateNeedsUpdate checks if a new launch template version is needed
```

Comment:
nit: s/launch/Launch

Reply:
not even a nit, that would fail the linter 🖖